rules/linux/persistence_rpm_package_installation_from_unusual_parent.toml

```diff
@@ -21,12 +21,16 @@ name = "RPM Package Installed by Unusual Parent Process"
 risk_score = 21
 rule_id = "8cc72fa3-70ae-4ea1-bee2-8e6aaf3c1fcf"
 setup = """## Setup
+
```

Comment:
I don't see any query adjustments here. If not the contents will still version bump the rule and display to the user that this rule has been updated.

Reply:
I reverted the change to that rule. I will fix the setup guide in a separate bulk PR.